core/domain/skill_jobs_one_off.py

```diff
@@ -116,3 +116,63 @@ def reduce(key, values):
                 sum(ast.literal_eval(v) for v in values))])
         else:
             yield (key, values)
+
+
+class SkillCommitCmdMigrationOneOffJob(jobs.BaseMapReduceOneOffJobManager):
```

Comment:
This need an infrastructural fix where we set up a proper system for update of existing commands: #10807